### PR TITLE
added e2e to verify remappings

### LINKED ISSUE #4235: E2E: use of remappings.txt
when no config file is created, when a hardhat or foundry env is detected.

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #4235

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/ballot.test.ts [hunk 1]
```diff
@@ -92,6 +92,25 @@ module.exports = {
...
+      // eslint-disable-next-line dot-notation
+      .execute(function () { document.querySelector('*[data-id="modalDialogCustomPromptTextCreate"]')['value'] = 'workspace_remix_default' })
+      .waitForElementPresent('[data-id="fileSystemModalDialogModalFooter-react"] .modal-ok')
+      .execute(function () { (document.querySelector('[data-id="fileSystemModalDialogModalFooter-react"] .modal-ok') as HTMLElement).click() })
+      .pause(1000)
+      .waitForElementVisible('*[data-id="treeViewLitreeViewItemcontracts"]')
+      .addFile('contracts/lib/storage/src/Storage.sol', { content: retrieverContract })
+      .addFile('remappings.txt', { content: 'storage=contracts/lib/storage/src' })
+      .addFile('contracts/Retreiver.sol', { content: storageContract })
+      .verifyContracts(['Retreiver', 'Storage'])
```
Comment (yann300) [inline apps/remix-ide-e2e/src/tests/ballot.test.ts:111]: typo, the test should actually fails.

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/ballot.test.ts:111]: no, remappings are defined, so it will pass.

Comment (yann300) [reply apps/remix-ide-e2e/src/tests/ballot.test.ts:111]: I mean you are checking `Retreiver` but the contract is named `Retriever`

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/ballot.test.ts]: Ah, fixed it.

**Review by yann300 — APPROVED** [review]

**Pull request merged**